plugins/outputs/nats/README.md

```diff
@@ -39,8 +39,19 @@ to use them.
   # credentials = "/etc/telegraf/nats.creds"
 
   ## NATS subject for producer messages
+  ## For jetstream this is also the subject where messages will be published
   subject = "telegraf"
 
+  ## Jetstream specific configuration. If specified, telegraf will use Jetstream to publish messages
+  # [outputs.nats.jetstream]
```

Comment:
Would a user write telegraf data to different streams? Or is it more likely that they have different outputs for different streams?

Reply:
Ideally telegraf should write to just one stream, if the use case requires multiple streams, then multiple output plugins would be defined.
I see why you may have asked this question- Is it the TOML syntax that looks confusing for the same?

Reply:
I needed to ask the question so we ensure we picked the right options at the start rather than needing to go back and update the plugin to support multiple streams.